articles/microsoft-2021-insights/components/inline-check.js

```diff
@@ -1,11 +1,16 @@
-import { Icons } from '@carbonplan/components'
+import { CheckCircle } from '@carbonplan/icons'
 
 const InlineCheck = () => {
```

Comment:
sure! good call

Reply:
i will flag, though, if we're looking at a potential future CMS world, or maybe just as good practice, it might be a useful constraint to say that each article either imports from public packages _or_ imports from components defined alongside within article... adding this one would create exceptions to that rule, but is that rule potentially important? i'm not sure

Reply:
Hmmm I guess I'm not sure what the benefit of making this a firm _rule_ rather than a norm would be? It seems like we'll always want the flexibility to develop components outside of a CMS editor, so we should always have some flexibility around where components are defined.

Reply:
fair enough! happy to make the change, agreed we can always revisit